src/default/assets/js/src/typedoc/utils/trottle.ts

```diff
@@ -0,0 +1,13 @@
+namespace typedoc
+{
+    export const throttle = <A extends any[]>(fn: (...args: A) => void, wait = 100) => {
```

Comment:
This has the potential for us to lose one call after the event has stopped being fired, but since it is called every 10ms anyways, I don't think it matters. I couldn't tell any difference when playing around with it.